### ISSUE TITLE
white board - must have

### ISSUE THREAD

**Issue by joaopequeno-git** [geovens/gInk#46]
hi,

can you implement a white board function?
that was great.
I think this project is great!

**Comment by geovens** [comment]
hi,

What is a white board function?

**Comment by joaopequeno-git** [comment]
Its a function that the epic pen have and others that turns the screen blank to you can write/draw what ever you want, as a note or similar....
Even the windows ink function have the white board for example.

**Comment by joaopequeno-git** [comment]
> 
> 
> hi,
> 
> What is a white board function?

Its a function that the epic pen have and others that turns the screen blank to you can write/draw what ever you want, as a note or similar....
Even the windows ink function have the white board for example.

**Comment by joaopequeno-git** [comment]
> hi,
> 
> What is a white board function?

Hello,

Have you consider this function?
The main advantage is to be drawing in a blank screen without any other stuff in the screen.

Thanks

**Comment by thfrei** [comment]
As a loving user of gInk, I think it is part of the main point to draw on top of the screen. 

If you would like to use whiteboard functionality you could just use OneNote or some other free MS tool (like whiteboard), this would also give you the advantage of having a bigger canvas and be able to draw more and scroll.

What do you think?

**Comment by pubpub-zz** [comment]
I've implemented this function in my fork https://github.com/pubpub-zz/ppInk. if you want to have a try.

**Comment by joaopequeno-git** [comment]
> 
> 
> As a loving user of gInk, I think it is part of the main point to draw on top of the screen.
> 
> If you would like to use whiteboard functionality you could just use OneNote or some other free MS tool (like whiteboard), this would also give you the advantage of having a bigger canvas and be able to draw more and scroll.
> 
> What do you think?

I agree, but the best part of this software was to have all in one, whats whould be the cherry on top of the cake.

**Comment by nunez0** [comment]
> I've implemented this function in my fork https://github.com/pubpub-zz/ppInk. if you want to have a try.

Hello folks!
This sw is awesome.
As a loyal and regular user, the whiteboard function can be very useful  when  we need to give a temporary highlight or explain something. Is not rarely to me during an on-line presentation someone asks for detailed information of a topic / or / asks to solve a calculation with more steps. For these cases, a option to make all screen be blank for a while is really useful.

**Comment by pubpub-zz** [comment]
I've introduced such a function in the fork I've developped: 
https://github.com/pubpub-zz/ppInk
You can change the background color when ever you with a long click on the bin, or set it more permanently through the options
